components/sensor/zyaura.rst

```diff
@@ -0,0 +1,84 @@
+ZyAura CO2 & Temperature & Humidity Sensor
+======================================
```

Comment:
See https://travis-ci.org/esphome/esphome-docs/builds/552679607#L503 - lint check is failing on this line

```suggestion
==========================================
```